requirements.txt

```diff
@@ -1,4 +1,4 @@
-retworkx>=0.11.0
+reseaux @ git+https://github.com/IvanIsCoding/retworkx@rename-retworkx
```

Comment:
Please change this to a regular requirement so we don't forget - Terra won't be merging this PR til the named package is released on PyPI anyway.

Reply:
The PR is WIP, I put this more in-place so we can actually get results from the CI rather than having import errors